### ISSUE TITLE
Goal 8.1: Fix validation dataset usage (training vs testing proportions)

### ISSUE THREAD

**Issue by yumna-tree** [bschink/pocket-narrator#24]
- [x] look into how training vs testing applies to text generation (bcs we know how it works on other types of predictions such as classification ML methods but here hehe)

-> ASK IN CLASS: which is better
A.) use the test set for normal evaluation immediately after training sessions and finetune based on results frequently-> risk: we glorify the test set and overfit unconsciously
B.) use it only at the end of the project -> risk: surprising scores which throw us off
